README.md

```diff
@@ -166,8 +175,28 @@ Example:
 github_workflow_usage_seconds{id="2862037",name="Create Release",node_id="MDg6V29ya2Zsb3cyODYyMDM3",repo="xxx/xxx",state="active",os="UBUNTU"} 706.609
 ```
 
+## Setting up authentication with GitHub API
+
+There are two ways for github-actions-exporter to authenticate with the GitHub API (only 1 can be configured at a time however):
+
+1. Using a GitHub App (not supported when you use Github Enterprise )
```

Comment:
I don't think is quite right? GitHub Enterprise is just a subscription https://github.com/pricing. Do you mean:

```
1. Using a GitHub App (not supported at the GitHub Enterprise management hierarchy level)
```